app/controllers/submissions_controller.rb

```diff
@@ -24,6 +27,13 @@ def create
 
   private
     def submission_params
-      params.require(:submission).permit(:user_id, :title, :description, :url, :screenshot)
+      params.require(:submission).permit(:title, :description, :url, :screenshot)
+    end
+
+    def check_logged_in
```

Comment:
Define this in your application controller instead of in each individual controller.